centml/sdk/api.py

```diff
@@ -31,7 +32,7 @@ def get(depl_type):
 
 def get_status(id):
     with get_api() as api:
-        return api.get_deployment_status_deployments_status_deployment_id_get(id)
+        return api.get_deployment_status_deployments_status_deployment_id_get(int(id))
```

Comment:
why do we need to cast to int? I think id is already an int

Reply:
yeah i just re checked its already coming in integer let me revert

Reply:
fixed